### PR TITLE
Doc 10314a

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -1,6 +1,236 @@
...
+
+
+=== Eventing Service
+
+[#table-known-issues-712-eventingservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52376[MB-52376^]
+| CI test for eventing multi collection
```
Comment (jon-strabala) [inline modules/release-notes/pages/relnotes.adoc]: @tonyjhillman This is an internal test - don't think customer need to see this MB

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -1,6 +1,236 @@
...
+| Unable to modify function settings when user has only eventing_manage_functions role
+
+| https://issues.couchbase.com/browse/MB-52587[MB-52587^]
+| Eventing Multi Collection: Inter function recursion not detected in case of sbm handler
+
+| https://issues.couchbase.com/browse/MB-52369[MB-52369^]
+| Eventing Service should honor the CPU & Memory limits set in cgroups
+
+| https://issues.couchbase.com/browse/MB-52373[MB-52373^]
+| Narrow UI's sometimes drop the Eventing service from the alternate menu
```
Comment (jon-strabala) [inline modules/release-notes/pages/relnotes.adoc]: @tonyjhillman This is not a BUG (UI people claim it was a feature) - no change was done so customers do not need to see this MB

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: Removed.

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: Removed.

**Pull request merged**